dm/dumpling/dumpling.go

```diff
@@ -222,9 +223,26 @@ func (m *Dumpling) Status(_ *binlog.SourceStatus) interface{} {
 		FinishedRows:      mid.FinishedRows,
 		EstimateTotalRows: mid.EstimateTotalRows,
 	}
+	go m.printStatus(*s)
```

Comment:
i forget why here need to use another thread to print, do you have any ideas?

Reply:
Maybe it could decouple users' querying from probable disk I/O (for log writing)? I don't think it does augment a significant performance though🤣

Reply:
syncer's `printStatus` needs lock thus maybe slow. No need to block RPC. Then I simply use same style for other units.

Reply:
I think it's OK we use another goroutine for this light-weighted work or not.